config/locales/in_person_proofing/es.yml

```diff
@@ -73,7 +73,6 @@ es:
         retail_hours_weekday: 'De Lunes a Viernes:'
         selection: 'Esta es la oficina que seleccionó:'
       prepare:
```

Comment:
Missing update to po_search_about, above. Proposed:
`po_search_about:  Puede verificar su identidad en persona en una oficina
            de correos local de los Estados Unidos.`

Reply:
Excellent catch Sonia! This change is being made to the content changes happening in [this PR](https://github.com/18F/identity-idp/pull/8312/files#r1181800142)